frontend/src/page/Login.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { signInWithPopup } from 'firebase/auth';
+import './login.css';
+import googlelogo from '../assets/google_logo.svg';
+import { auth, provider } from '../services/firebase';
+import { getUser } from '../api/user';
+
+function Login() {
+  const [userName, setUserName] = useState('');
+  const navigate = useNavigate();
+  const [username, usernameToggle] = useState(false);
+
+  const googleLogin = async () => {
+    signInWithPopup(auth, provider);
+    const res = await getUser(userName);
+    if (res.error === 'Username exists') {
+      navigate('/');
+    } else {
+      usernameToggle(!username);
+    }
+  };
+
+  return (
+    <div>
+      <h1>Log in to Aimer!</h1>
+      <div className="inner-box">
+        <form>
```

Comment:
The code for the form looks almost the exact same as the Signup form. Make this a component that can be used in both the `Signup` and `Login` pages. 

If you don't know what a component is go through and do the React tutorial and read and try to understand everything:
https://react.dev/learn/tutorial-tic-tac-toe

Reply:
Note: If you ever notice you're copying/pasting the same code everywhere you can separate that into its own component or function.